<ui version="4.0" >
 <class>ImageInfoPanelWdg</class>
 <widget class="QFrame" name="ImageInfoPanelWdg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>116</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="frameShape" >
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>3</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="m_pThumbL" >
     <property name="text" >
      <string>Thumb</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="m_pRightSideW" >
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="m_pPosL" >
        <property name="text" >
         <string>Pos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_pDimL" >
        <property name="text" >
         <string>Dim</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_pSizeL" >
        <property name="text" >
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>49</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_pFullB" >
        <property name="minimumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>View full-size image</string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="Mp3Diags.qrc" >:/images/zoom-in.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" native="1" name="widget" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_pAssignB" >
           <property name="minimumSize" >
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Assign picture to songs</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="Mp3Diags.qrc" >:/images/assign.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_pEraseB" >
           <property name="minimumSize" >
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Erase local file</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="Mp3Diags.qrc" >:/images/remove_file.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Mp3Diags.qrc" />
 </resources>
 <connections/>
</ui>
